src/sentry/static/sentry/app/bootstrap.tsx

```diff
@@ -139,6 +139,34 @@ const render = (Component: React.ComponentType) => {
   }
 };
 
+function changeFavicon(theme: 'dark' | 'light'): void {
+  const n = document.querySelector<HTMLLinkElement>('.js-site-favicon[type="image/png"]');
```

Comment:
do you need the class name? would `[rel="icon"][type="image/png"]` suffice?

Reply:
not going to lie... I stole this directly from github including the class name.

Reply:
shortened